### DISCUSSION TITLE
Can't renew my GitHub Student Developer Pack.

### DISCUSSION THREAD

**Discussion by brentbaylon** [community/community#59356] [GitHub Education]
I recently received the email which says they can't renew my subscription because I don't have the school email  to use with the subscription. However, I clearly stated in the form that our school doesn't provide us (students) school emails and only encourages us to create a Gmail account for educational purposes only.

Also last year, I successfully applied for the GitHub Student Developer Pack using my Gmail account and not the school email (since the school doesn't provide us) without any problems.

**Comment by samus-aran** [comment]
Hi @AnonymouseHucker29  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

It would also be recommended that either you or a faculty member provides evidence that students do not receive academic emails to help with the request. 👍

Hope this helps! Let us know how you get along 👍
